components/textbook-demo/LearningPathPagesSection.vue

```diff
@@ -52,7 +52,7 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
-import { SegmentData } from '~/constants/appLinks'
+import { SegmentData, textbookDemoIntroductionCourseStartLearningUrl } from '~/constants/appLinks'
```

Comment:
How do you think if we create a constants file on a textbook-demo folder to add the urls we are going to reuse? Right now we only have one, but if we continue developing learning path we will have more

Reply:
I like the idea in general, but I have some concerns about creating a new pattern for one single use case (as for now), because there is potential of forgetting that pattern later.

After some more thinking, I wanted to suggest defining the URL in the parent component (`pages/textbook-demo/learning-paths/introduction-course.vue`) and passing it as a prop named something similar to `courseStartUrl`, which may be generic enough to make sense for both components, and also for future path pages that may reuse this template.

What do you think? 🙂

Reply:
As discussed offline, we'll move the URL to appLinks until we find a global way of handling the links with https://github.com/Qiskit/qiskit.org/issues/1566.